### PR TITLE
Feature/group 9 questions

### PR DESCRIPTION
**Pull request by Blessed0314**

### CODE REVIEW

File: public/data/0828614-2724/info.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "user": {
+    "username": "leon.dominguez",
+    "profileImage": "https://media.licdn.com/dms/image/v2/D4E03AQFCzbUThqmePA/profile-displayphoto-shrink_200_200/profile-displayphoto-shrink_200_200/0/1729999557983?e=1735776000&v=beta&t=yk_Rvb-B9bJWV1ySaHBf_v6jQwod_OrifpSBNBPeo8A",
+    "socialLink": "@leondominguezg"
+  },
+  "topics": [
+    {
+      "topic": "locations and common places",
```
Comment (fredyballest) [inline public/data/0828614-2724/info.json]: Not good!!  The topic should be grammar or vocabulary.

File: public/data/2128732-2724/info.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+    "user": {
+      "username": "zaira-navia",
+      "profileImage": "https://lh3.googleusercontent.com/a/ACg8ocIX55L8QIoQkrv9FQCFbLIB8GsiyO8okefMwJ0My8Slgzc9tNbh=s96-c-rg-br100",
+      "socialLink": "@adz33"
+    },
+    "topics": [
+      {
+        "topic": "transportation",
```
Comment (fredyballest) [inline public/data/2128732-2724/info.json]: Not good!!  The topic should be grammar or vocabulary.

File: public/data/2128732-2724/info.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          },
+          {
+            "question": "Where would you find a taxi?",
+            "options": ["at the taxi stand", "in the taxi stand", "on the taxi stand", "the taxi stand"],
+            "answer": "at the taxi stand"
+          }
+        ]
+      },
+      {
+        "topic": "daily routines",
```
Comment (fredyballest) [inline public/data/2128732-2724/info.json]: Not good!!  The topic should be grammar or vocabulary.

**Comment by fredyballest** [comment]
Almost have it. Please review the comments.

**Review by fredyballest — CHANGES_REQUESTED** [review]
Please check the reviews

**Review by fredyballest — CHANGES_REQUESTED** [review]
Please check the comments

**Review by fredyballest — CHANGES_REQUESTED** [review]
Not good!! The topic should be grammar or vocabulary.

**Review by fredyunivalle — APPROVED** [review]
LGTM

**Review by fredyballest — APPROVED** [review]

**Pull request merged**